gulpfile.js

```diff
@@ -82,6 +83,66 @@ function createReloadServer() {
   });
 }
 
+function writeServiceWorkerFile() {
+  let path = require('path');
+  let rootDir = 'dist';
+
+  let config = {
+    cacheId: 'polymerproject',
+    staticFileGlobs: [
+      `${rootDir}/1.0/index.html`,
+      `${rootDir}/1.0/shell`,
+      `${rootDir}/1.0/about.html`,
+      `${rootDir}/elements/**`,
+      `${rootDir}/js/*.js`,
+      `${rootDir}/css/*.css`,
+      `${rootDir}/bower_components/**/webcomponents-lite.min.js`,
+      `${rootDir}/bower_components/highlight/highlight.js.js`,
+    ],
+    dynamicUrlToDependencies: {
+      '/1.0/shell.html': [
```

Comment:
I think the property name should be `/1.0/shell`, not `/1.0/shell.html`. The property names are meant to be the actual URLs that the server knows about, which doesn't include `.html`.